license-upgrade/core-traditional-unbreak.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+## Script to automatically determine what version of plugin needs to be downloaded
+## and installs it
+
+hinit() {
+    rm -f /tmp/hashmap.$1
+}
+
+hput() {
+    echo "$2 $3" >> /tmp/hashmap.$1
+}
+
+hget() {
+    PLUGIN_URL=$(grep "^$2 " /tmp/hashmap.$1 | awk '{ print $2 };' )
+}
+
+verify_command() {
+  echo "Verifying command [${1}] is installed..."
+
+  if command --version $1 >/dev/null 2>&1 || command -version $1 >/dev/null 2>&1 || command -v $1 >/dev/null 2>&1; then
```

Comment:
Wouldn't be easier to use `which`?

Reply:
AFAICT it's a kernel built-in one, so granted to be available.

Reply:
on a mac its an actual program apparently

Reply:
so i did some reading, and for bash, it seems `command` is the preferred way to do it
https://unix.stackexchange.com/questions/85249/why-not-use-which-what-to-use-then/85250#85250